cluster/saltbase/salt/etcd/etcd.manifest

```diff
@@ -39,7 +39,7 @@
 "containers":[
     {
     "name": "etcd-container",
-    "image": "gcr.io/google_containers/etcd:{{ pillar.get('etcd_docker_tag', '2.2.1') }}",
+    "image": "gcr.io/google_containers/etcd:{{ pillar.get('etcd_docker_tag', '3.0.14-alpha.1') }}",
```

Comment:
I wanted to be consistent with versioning scheme for k8s itself, where we are doing "-alpha.1 -alpha.2, ... -beta.1, -beta.2, ...".

I don't think it is that important though, so leaving as is (we can change it later if you feel strongly about it).